org/responses.py

```diff
@@ -22,3 +22,23 @@
 user_already_present_409 = {
   "message": "Already a member of organization"
 }
+
+user_unauthorized_401 = {
+  "detail": "Authentication credentials were not provided."
```

Comment:
Is there any specification, where you use `detail` and `message` (Like anything which you guys are following)?

Reply:
Nice catch @BadduCoder ! I think since drf token auth returns `detail` in its response for 401. We can follow the same standard for our custom errors. 
```
{
  "detail": "You are not a member of this organisation"
}
```

Reply:
Yeah, we can use detail for all responses. this can be a good first issue.